src/olympia/amo/templatetags/jinja_helpers.py

```diff
@@ -447,3 +451,20 @@ def format_datetime(value, format='DATETIME_FORMAT'):
 def class_selected(a, b):
     """Return ``'class="selected"'`` if ``a == b``."""
     return mark_safe('class="selected"' if a == b else '')
+
+
+class Spaceless(Extension):
```

Comment:
A couple basic tests covering this extension in `src/olympia/amo/tests/test_helpers.py` would be nice

Reply:
https://github.com/mozilla/addons-server/pull/17354/commits/a0e7556a6055449ec35da523b24114605c266c67 its just one test, but I tried to cover what it does and the edge cases.